### PR TITLE
fix: helm install error

### LINKED ISSUE #2589: helm install error（when i use pvcs，it occured）
## Bug Report

**What version of Kubernetes are you using?**
1.19.8
Actually，it will occur at any version

**What version of Chaos Mesh are you using?**
2.1.0

**What did you do?**
when i change pvc config ，helm install failed
当我开启helm脚本中的pvc配置时，将会出现此问题
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: apiVersion not set
image

use "helm template -f values.yaml . >deployChart.yaml",i found this annotation in deployChart.yaml
使用helm template -f values.yaml . >deployChart.yaml查看deployChart.yaml，发现apiVersion字段被注释掉了
image

Compared with the previous version, it seems to be caused by the license statement in the chaos-dashboard-pvc.yaml file, but I tried to modify it and it couldn’t be restored. Finally, I had to delete all the licenses and restore them to normal.
对比了一下之前的版本，貌似是由于chaos-dashboard-pvc.yaml文件中的license声明导致的，但我尝试修改也不能恢复，最后只能把license全部删除后恢复正常

**What did you expect to see?**
helm install success
helm安装成功

**What did you see instead?**

**Output of chaosctl**

### PR DESCRIPTION
**Pull request by kxzonly**
### What problem does this PR solve?

Close #2589

> Problem Summary:

### What's changed and how it works?

> What's Changed:

helm/chaos-mesh/templates/chaos-dashboard-pvc.yaml

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Chaos Dashboard`
- [x] Need to **cheery-pick to release branches**

### Checklist

Tests

- [x] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
fix issue #2589 and changed files: helm/chaos-mesh/templates/chaos-dashboard-pvc.yaml
```

### CODE REVIEW

File: pkg/ctrlserver/graph/generated/generated.go [hunk 1]
```diff
@@ -14,13 +14,12 @@ import (
 
 	"github.com/99designs/gqlgen/graphql"
 	"github.com/99designs/gqlgen/graphql/introspection"
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
```
Comment (cwen0) [inline pkg/ctrlserver/graph/generated/generated.go:17]: Can you revert these changes?

Comment (kxzonly) [reply pkg/ctrlserver/graph/generated/generated.go:17]: OK，i tried it，please wait a moment

Comment (kxzonly) [reply pkg/ctrlserver/graph/generated/generated.go]: finished it ,i restored generate file changes

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by g1eny0ung — APPROVED** [review]

**Review by STRRL — APPROVED** [review]
LGTM! Thanks for your contribution!

**Comment by STRRL** [comment]
> cherry pick to release-2.0 failed

We do not apply skywalking license checker and license boilerplate on branch release-2.0, so we do not cherry-pick it on release-2.0. I am going to remove this tag.

**Pull request merged**